package.json

```diff
@@ -14,21 +14,21 @@
     "README.md"
   ],
   "sideEffects": false,
-  "directories": {
-    "test": "test"
-  },
   "scripts": {
     "build": "rollup --config .rollup.js",
     "prepublishOnly": "npm run build",
     "release": "np",
-    "test": "eslint '{index,lib/**/*,test/**/*}.js' && mocha --require esm 'test/*.js' --delay",
+    "test": "jest --silent",
     "version": "version-changelog CHANGELOG.md && changelog-verify CHANGELOG.md && git add CHANGELOG.md"
   },
   "dependencies": {
-    "postcss-query-ast": "^1.0.0",
-    "stylelint": "^9.9.0"
+    "postcss-query-ast": "^2.0.0",
+    "stylelint": "^13.13.1"
```

Comment:
@vyobukhov this could be peer dependancy since Stylelint plugin can’t work without Stylelint anyways